### ISSUE TITLE
add high speed SFP link

### ISSUE THREAD

**Issue by gkasprow** [sinara-hw/EEM_FMC_Carrier#60]
several users asked us for high speed SFP link
we can do it as a mezzanine and SATA connector
we can easily fit SFP and route clk in as a pigtail
![obraz](https://github.com/user-attachments/assets/b34bf02f-356e-4de0-8fad-06b5d6b59bc1)

**Comment by marmeladapk** [comment]
Should I remove the front panel clock signal completely since we have internal mmcx on the back or leave this signal as mmcx near the front?

**Comment by marmeladapk** [comment]
Also, currently we have 4 GTP lanes available on this FPGA. In EEM variant all are routed to FMC. In DIOT variant 1 of those can be rerouted to the backplane. Should I add another mux for this signal?

**Comment by gkasprow** [comment]
> Should I remove the front panel clock signal completely since we have internal mmcx on the back or leave this signal as mmcx near the front?

yes, one mmcx is enough; we can always use SMA pigtail to route it to the panel

**Comment by marmeladapk** [comment]
I have mixed feelings about using assembly option here. It would either mean keeping two stocks of PCBs or sometimes expecting users to solder 0201 capacitors. Both are unrealistic IMO.

I feel like the following connections are a better choice, since these muxes are quite cheap.

```
FPGA MGT2 --- MUX --- SFP
               |----- FMC MGT 2

FPGA MGT3 --- MUX --- BP MGT
               |----- FMC MGT 3
```

For comparison, this is one possible diagram with a capacitor assembly option.
```
FPGA MGT3 --- MUX --- BP MGT
               |------------- Capacitors --- SFP
                                   |-------- FMC MGT 3
```
